mapper/mapper.go

```diff
@@ -16,5 +18,15 @@ func AllDatasets(datasets dataset.List) []model.Dataset {
 			Title: ds.Next.Title,
 		})
 	}
+
+	sort.Slice(mappedDatasets, func(i, j int) bool {
+		if mappedDatasets[i].Title == "" {
+			return false
+		} else if mappedDatasets[j].Title == "" {
```

Comment:
Nothing wrong with this but you can remove the `else` and have another `if` statement as the func will return if the first case is true. Just makes the code a little less cluttered.